angular.json

```diff
@@ -67,33 +61,23 @@
         "test": {
           "builder": "@angular-devkit/build-angular:karma",
           "options": {
-            "main": "projects/ngx-mask-lib/src/test.ts",
-            "karmaConfig": "projects/ngx-mask-lib/karma.conf.js",
-            "tsConfig": "projects/ngx-mask-lib/tsconfig.spec.json",
+            "main": "src/test.ts",
+            "karmaConfig": "src/karma.conf.js",
+            "polyfills": "src/polyfills.ts",
```

Comment:
@ColinMorris83  I was curious about this when I was fiddling around with the unit tests - do we need polyfills for ChromeHeadless? Or are they kept just in case someone wants to run the tests in a different browser?

Reply:
Yes they are needed as they do the import of 'zone.js/dist/zone';
Without it you will get ReferenceError: Zone is not defined

Reply:
Interesting - when I had removed it the full test suite still ran successfully on my Mac. Thank you!

Reply:
That's because you had changed it to point at the test.ts inside the projects folder, and that file already does an import of zone.js/dist/zone.